examples/logger.py

```diff
@@ -3,13 +3,38 @@
 """
 
 import time
+import logging
 from typing import Any, Optional, Tuple
 
 from PyQt5.QtCore import QObject, pyqtSlot, pyqtSignal
 from PyQt5.QtWidgets import QWidget, QVBoxLayout, QPushButton, QTextEdit, QLabel, QDialogButtonBox
 
 from qiwis import BaseApp
 
+class Signaller(QObject):
+    """Signal for LoggingHandler"""
+    signal = pyqtSignal(str)
+
+class LoggingHandler(QObject, logging.Handler):
+    """Handler for logger.
+
+    Sends log message to connected function using emit
+
+    Signal:
+          signal: Helps to send message to destination.
```

Comment:
Where is the signal used? I wonder what you intended.

Reply:
Good review. I should delete it.